spec/fixtures/generate_crt_and_Key_for_1028_keylength.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+mkdir ca && cd ca
+if [ $? -ne 0 ]; then
+   exit 1
+fi
+
+# 1. Creating a private key  
+openssl genrsa -out key_for_weak_key_cert.pem 1024
+
+# 2. Creating a certificate request 
+openssl req -verbose -new -key key_for_weak_key_cert.pem -out ca.csr  -subj "/C=US/ST=WA/L=Seattle/O=SUSE/OU=CaaSP/CN=1024 key length"
+
+# 3. Generating an openssl environment in the working directory (where the files are located at with SUSE OS)
+mkdir -p ./demoCA/newcerts; touch ./demoCA/index.txt; touch ./demoCA/index.txt.attr ;echo 01 > ./demoCA/serial
```

Comment:
Please split up these lines. Using semicolon to put all the commands on one line just makes it messy.